beachball.config.js

```diff
@@ -18,6 +18,8 @@ module.exports = {
       if (name === '@fluentui-react-native/dependency-profiles') {
         console.log(`Updating ${name} to use latest published versions`);
         execSync(`yarn update-profile`, { cwd: packagePath });
+        console.log('Updating lockfile');
+        execSync(`yarn install  --mode update-lockfile`);
       }
     },
```

Comment:
Note: This will only update if `@fluentui-react-native/dependency-profiles` is updated

Reply:
This should be ok since the hook is called after bump has been run on all packages.